### ISSUE TITLE
Select All in "Project Selection" filter

### ISSUE THREAD

**Issue by joachimcarrein** [cribeiro84/azure-devops-pull-request-hub#206]
**Is your feature request related to a problem? Please describe.**
We have multiple projects, if I want to select all, I need to click the first, the the dropdown closes and the filter refreshes and I need to repeat.

This would allow to select all and deselect a single few.

**Describe the solution you'd like**
A button to select all entries, or when none selected show all.

**Describe alternatives you've considered**
Deselecting all projects
It seems when pressing clear all, it just remembered my previous project filter and did not show all projects

**Comment by miai-demant** [comment]
'Select all' projects is most important needed-feature at this moment. Please add it.

We have hundreds projects in our Azure DevOps and I might be contributor in many of them.

Thank You

**Comment by LePtitJoNadeau** [comment]
I agree. The save current filter isn't working that good. I often need to select my projects manually.  A button where it's select all projects would be nice

That would look like something like that
![image](https://user-images.githubusercontent.com/23142259/208309526-c1e2dc93-e578-42d2-99af-6bc3ba9b8fee.png)
